src/template-tags/tickets.php

```diff
@@ -192,20 +193,24 @@ function tribe_events_count_available_tickets( $event = null ) {
 	function tribe_tickets_buy_button( $echo = true ) {
 		$event_id = get_the_ID();
 
-		// check if there are any tickets on sale
-		if ( ! tribe_events_has_tickets_on_sale( $event_id ) ) {
-			return null;
+		if ( empty( $event_id ) ) {
+			return '';
+		}
+
+		// check if there are any tickets available
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 05623631a75c14eacc1a1b5f4c8d5569</sub>

Reply:
```suggestion
		// Check if there are any tickets available.
```